src/transformers/modeling_utils.py

```diff
@@ -117,6 +117,7 @@
         save_offload_index,
         set_module_tensor_to_device,
     )
+    from accelerate.utils.modeling import get_state_dict_from_offload
```

Comment:
You need to protect the import since we don't force the users to download the latest version of accelerate. You can see how we can do [here](https://github.com/huggingface/transformers/pull/31052/files#:~:text=if%20quanto_version%20%3E%3D,qint2%2C%20qint4).